pybamm/solvers/processed_variable.py

```diff
@@ -386,9 +437,9 @@ def __call__(self, t=None, x=None, r=None, y=None, z=None, warn=True):
         # an error
         if t is None:
             if len(self.t_sol) == 1:
-                t = self.t_sol
+                t = self.t_sol * self.timescale
             elif self.base_variable.is_constant():
-                t = self.t_sol[0]
+                t = self.t_sol[0] * self.timescale
```

Comment:
Am wondering whether `t_sol` should be stored in seconds. I guess you are either consistent with `sol.t`, or consistent with `first_dim_pts`, but you can't have both

Reply:
Yeah I went back and forth on this. I've added an attribute `t_pts` which is dimensional time. That way things ending `_sol` are the dimensionless values and things ending `_pts` are dimensional (and are the values used in the interpolant)

Reply:
Great, that makes sense!